src/components/component-playground.jsx

```diff
@@ -463,5 +479,35 @@ module.exports = React.createClass({
       orientation: contentNode.offsetHeight > contentNode.offsetWidth ?
                    'portrait' : 'landscape'
     });
+  },
+
+  _getFilteredFixtures() {
+    var components = this.props.components;
+
+    return _.reduce(components, function(acc, component, componentName) {
+      var fixtureNames = Object.keys(component.fixtures);
+      var filteredFixtures = _.filter(fixtureNames, function(fixtureName) {
+        return fixtureName.indexOf(this.state.searchText) !== -1 ||
+               this._isCurrentFixtureSelected(componentName, fixtureName);
```

Comment:
Cool idea! I'd add a comment to specify that I'm showing the currently selected fixture even if it doesn't match the current search query. This is what happens, right?